contracts/EntityFundingFacet.sol

```diff
@@ -8,6 +8,7 @@ import "./base/IDiamondFacet.sol";
 import "./base/IERC20.sol";
 import "./base/IMarket.sol";
 import "./EntityFacetBase.sol";
+import "hardhat/console.sol";
```

Comment:
should remove test tooling prior to live deployments

Reply:
Yes, of course, I though I did remove it 🤦🏻‍♂️

Reply:
done